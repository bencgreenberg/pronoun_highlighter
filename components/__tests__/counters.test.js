import React from 'react';
import Counter from '../counters';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Counter femaleNum={1} maleNum={2} neutralNum={3} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});