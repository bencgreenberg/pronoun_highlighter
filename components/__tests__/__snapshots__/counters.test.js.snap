// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="jsx-725514212 textBox"
>
  <div
    className="jsx-725514212 counter-area"
  >
    Female Count: 
    1
  </div>
  <div
    className="jsx-725514212 counter-area"
  >
    Male Count: 
    2
  </div>
  <div
    className="jsx-725514212 counter-area"
  >
    Neutral Count: 
    3
  </div>
</div>
`;
